src/generator/AutoRest.CSharp.Azure/Templates/AzureMethodGroupTemplate.cshtml

```diff
@@ -5,42 +5,31 @@
 @EmptyLine
 namespace @Settings.Namespace
 {
-    using System;
     using System.Linq;
```

Comment:
Ah, yeah. 

The only option at that point would be to transform the call to a static method invocation. 

Which is kinda sucky, but not the end of the world.